client/scripts/views/components/login_with_wallet_dropdown.ts

```diff
@@ -56,17 +54,17 @@ const LoginWithWalletDropdown: m.Component<{
                   && m.route.param('prev') !== '/')
       ? m.route.param('prev')
       : joiningChain ? `/${joiningChain}`
-        : joiningCommunity ? `/${joiningCommunity}`
           : m.route.get().indexOf('web3login') === -1 && m.route.get().replace(/\?.*/, '') !== '/' ? m.route.get()
             : app.chain ? `/${app.chain.meta.chain.id}`
-              : app.community ? `/${app.community.meta.id}`
                 : '/?';
     // only redirect to home as an absolute last resort
 
-    const targetCommunity = app.community?.id;
-
-    const web3loginParams = loggingInWithAddress ? { prev, loggingInWithAddress, targetCommunity } : joiningChain
-      ? { prev, joiningChain } : joiningCommunity ? { prev, joiningCommunity } : { prev };
+    // TODO: double check this
```

Comment:
Let's be sure to QA this in particular.

Reply:
QA'd: tested going to a community page, logging out, and successfully logging back into the community page.